CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
```

Comment:
The wording still sounds like an improvement. Why is it marked as a fix?

Reply:
So it can stay a patch release. I can change it to improvement and update the version to a minor?

Reply:
I see. Please keep it as it.